Latest pulp/pulp:3.15 image does not work

It's impossible to upgrade / start a fresh container:
```
[2021-09-01 16:58:18 +0000] [524] [INFO] Worker exiting (pid: 524)
[2021-09-01 16:58:18 +0000] [469] [INFO] Shutting down: Master
[2021-09-01 16:58:18 +0000] [469] [INFO] Reason: Worker failed to boot.
[2021-09-01 16:58:18 +0000] [525] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/gunicorn/arbiter.py", line 589, in spawn_worker
    worker.init_process()
  File "/usr/local/lib64/python3.8/site-packages/aiohttp/worker.py", line 51, in init_process
    super().init_process()
  File "/usr/local/lib/python3.8/site-packages/gunicorn/workers/base.py", line 134, in init_process
    self.load_wsgi()
  File "/usr/local/lib/python3.8/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/usr/local/lib/python3.8/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/usr/local/lib/python3.8/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
    return self.load_wsgiapp()
  File "/usr/local/lib/python3.8/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/local/lib/python3.8/site-packages/gunicorn/util.py", line 359, in import_app
    mod = importlib.import_module(module)
  File "/usr/lib64/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/usr/local/lib/python3.8/site-packages/pulpcore/content/__init__.py", line 18, in <module>
    django.setup()
  File "/usr/local/lib/python3.8/site-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/usr/local/lib/python3.8/site-packages/django/apps/registry.py", line 114, in populate
    app_config.import_models()
  File "/usr/local/lib/python3.8/site-packages/django/apps/config.py", line 301, in import_models
    self.models_module = import_module(models_module_name)
  File "/usr/lib64/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/usr/local/lib/python3.8/site-packages/pulp_maven/app/models.py", line 7, in <module>
    from pulpcore.plugin.models import Content, Remote, Repository, BaseDistribution
ImportError: cannot import name 'BaseDistribution' from 'pulpcore.plugin.models' (/usr/local/lib/python3.8/site-packages/pulpcore/plugin/models/__init__.py)
[2021-09-01 16:58:18 +0000] [525] [INFO] Worker exiting (pid: 525)
[2021-09-01 16:58:18 +0000] [520] [WARNING] Worker with pid 525 was terminated due to signal 15
```

--- a/pulp/Containerfile
+++ b/pulp/Containerfile
@@ -18,7 +18,7 @@ RUN pip3 install --upgrade \
   pulp-container${PULP_CONTAINER_VERSION} \
   pulp-deb${PULP_DEB_VERSION} \
   pulp-file${PULP_FILE_VERSION} \
-  pulp-maven${PULP_MAVEN_VERSION} \
+  #pulp-maven${PULP_MAVEN_VERSION} \
   pulp-python${PULP_PYTHON_VERSION} \
   pulp-rpm${PULP_RPM_VERSION} \
   requests

--- a/pulp_nightly/Containerfile
+++ b/pulp_nightly/Containerfile
@@ -8,7 +8,7 @@ RUN pip3 install --upgrade \
   git+https://github.com/pulp/pulp_container@master \
   git+https://github.com/pulp/pulp_deb@main \
   git+https://github.com/pulp/pulp_file@main \
-  git+https://github.com/pulp/pulp_maven@master \
+  #git+https://github.com/pulp/pulp_maven@master \
   git+https://github.com/pulp/pulp_python@master \
   git+https://github.com/pulp/pulp_rpm@master \
   requests

